environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -684,6 +664,7 @@ objects:
               path: /-/ready
               port: 10902
               scheme: HTTP
+            initialDelaySeconds: 120
```

Comment:
@bwplotka We observed that indexing takes ~90 seconds for us right now. Is this too much?  What do you think?

Reply:
no - it's quite quick (: It can take few minutes (10-15min) easily for larger buckets

Reply:
Removing it for now, after offline discussions.